bot.go

```diff
@@ -130,6 +151,28 @@ func NewBot(confReader io.Reader) (*Bot, error) {
 	return b, nil
 }
 
+func (b *Bot) setupInfluxDb() error {
+	// Set up InfluxDB logging
+	err := b.Config("influxdb", &b.influxDbConfig)
+	if err == nil {
+		b.points = make(chan *client.Point, b.influxDbConfig.BufferSize)
+		b.influxDbClient, err = client.NewHTTPClient(client.HTTPConfig{
+			Addr:     b.influxDbConfig.URL,
+			Username: b.influxDbConfig.Username,
+			Password: b.influxDbConfig.Password,
+		})
+
+		if err != nil {
+			return err
+		}
+	} else {
```

Comment:
Should the err be logged here? Also, this could be cleaner with an `if err != nil { do stuff then return } everything else` rather than an if-else. Returning early is usually clearer.